### PR TITLE
use transformable_markers for kiva_pod

### PR DESCRIPTION
**Pull request by knorth55**
for `jsk_2016_01_baxter_apc`

### CODE REVIEW

**Comment by knorth55** [comment]
updated

File: jsk_2016_01_baxter_apc/euslisp/lib/baxterrgv5-interface.l [hunk 1]
```diff
@@ -393,7 +393,7 @@
           (setq box-list (send box-msg :boxes))
           (setq bin-list (list :a :b :c :d :e :f :g :h :i :j :k :l))
           (dolist (bin bin-list)
-            (setf (gethash bin _bin-boxes) (car box-list))
+            (sethash bin _bin-boxes- (car box-list))
```
Comment (wkentaro) [inline jsk_2016_01_baxter_apc/euslisp/lib/baxterrgv5-interface.l]: must be `_bin-boxes`, right?

Comment (wkentaro) [inline jsk_2016_01_baxter_apc/euslisp/lib/baxterrgv5-interface.l]: @pazeshun Could you please the naming of slots in baxter-rgv5.l?

Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/lib/baxterrgv5-interface.l]: Sorry for late response.
I'll change this after this PR.

**Pull request merged**